### PR TITLE
Add LeakRate quantity

### PR DESCRIPTION
**Pull request by debendda**

### CODE REVIEW

File: Common/UnitDefinitions/LeakRate.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+    "Name": "LeakRate",
+    "BaseUnit": "PascalQubicMeterPerSecond",
+    "XmlDocSummary": "The leakrate is a measure of the units of volume or mass leaking from a body.",
```
Comment (angularsen) [inline Common/UnitDefinitions/LeakRate.json:4]: I don't quite understand this description. Is there an alternative way to describe it?

From another source: https://www.leybold.com/en/knowledge/vacuum-fundamentals/leak-detection/definition-and-measurement-of-vacuum-leaks

> A leak rate of QL = 1 mbar · l/s is present when in an enclosed, evacuated vessel with a volume of 1 l the pressure rises by 1 mbar per second or, where there is positive pressure in the container, pressure drops by 1 mbar.

File: Common/UnitDefinitions/LeakRate.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{
+    "Name": "LeakRate",
+    "BaseUnit": "PascalQubicMeterPerSecond",
```
Comment (angularsen) [inline Common/UnitDefinitions/LeakRate.json:3]: `Qubic` should be `Cubic`

File: Common/UnitDefinitions/LeakRate.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    "XmlDocSummary": "The leakrate is a measure of the units of volume or mass leaking from a body.",
+    "BaseDimensions": {
+      "M": 1,
+      "L": 2,
+      "T": -3
+    },
+    "Units": [
+        {
+            "SingularName": "PascalQubicMeterPerSecond",
+            "PluralName": "PascalQubicMeterPerSecond",
```
Comment (angularsen) [inline Common/UnitDefinitions/LeakRate.json:13]: Meters plural

File: Common/UnitDefinitions/LeakRate.json [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+            "FromUnitToBaseFunc": "{x}",
+            "FromBaseToUnitFunc": "{x}",
+            "XmlDocRemarks": "https://de.wikipedia.org/wiki/Leckrate#Formelzeichen_und_Ma%C3%9Feinheit",
```
Comment (angularsen) [inline Common/UnitDefinitions/LeakRate.json:16]: Are there English references we can link to? I didn't find an English wiki though.

File: UnitsNet.Tests/CustomCode/LeakRateTests.cs [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+using System;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class LeakRateTests : LeakRateTestsBase
+    {
+        protected override double PascalQubicMeterPerSecondInOnePascalQubicMeterPerSecond => 1;
+
+        protected override double MillibarLiterPerSecondInOnePascalQubicMeterPerSecond => 10;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/LeakRateTests.cs:28]: Verified
https://www.sensorsone.com/leak-rate-converter/

File: UnitsNet.Tests/CustomCode/LeakRateTests.cs [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+        protected override bool SupportsSIUnitSystem => false;
+
+        protected override double TorrLiterPerSecondInOnePascalQubicMeterPerSecond => 7.5;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/LeakRateTests.cs:32]: This converter says `7.50061683`?
https://www.sensorsone.com/leak-rate-converter/

Comment (angularsen) [reply UnitsNet.Tests/CustomCode/LeakRateTests.cs:32]: This one says 7.5
https://www.helium-leak.eu/calculations-and-units/

Comment (angularsen) [reply UnitsNet.Tests/CustomCode/LeakRateTests.cs:32]: 7.5 here too
https://www.leybold.com/en-in/knowledge/vacuum-fundamentals/leak-detection/definition-and-measurement-of-vacuum-leaks

**Review by angularsen — CHANGES_REQUESTED** [review]
Hi and thank you. This looks almost good to go, a few minor comments.

**Comment by debendda** [comment]
Hi, thanks for the comments. I have improved all the suggestions. Unfortunately, I can't find an English wiki link either. I have added another link.

**Comment by angularsen** [comment]
Looks perfect, thanks!

Nuget should be out shortly.

[Release UnitsNet/5.8.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F5.8.0)

**Pull request merged**